### PR TITLE
Test command - more checks

### PR DESCRIPTION
**Pull request by tslashd**
Do not test  the `Map-Challenge` webhook if the plugin is not running on the server

### CODE REVIEW

**Comment by tslashd** [comment]
Along with what is mentioned at the top of the PR, I also added some more `logging` if webhook execution fails.

This is what `Discord` replies with if something is wrong:
image

So we try to get the `message` and `code` from it to be sent to server console. **_(UNTESTED)_**
In case of `Invalid Webhook` clients will be able to fix the issue much easier without a lot of pondering around.

File: scripting/SurfTimer-discord.sp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #include <mapchallenge>
 #pragma newdecls required
 #pragma semicolon 1
+// #define DEBUG // Enable this line and compile again to enable DEBUG messages
 
 public Plugin myinfo =
 {
```
Comment (Sarrus1) [inline scripting/SurfTimer-discord.sp:13]: Can you please also change the version to 2.5.7 so that I can trigger the build? Thanks!

**Review by Sarrus1 — CHANGES_REQUESTED** [review]

File: scripting/SurfTimer-discord.sp (hunk 1 above)
Comment (tslashd) [reply scripting/SurfTimer-discord.sp:13]: aye aye

**Review by Sarrus1 — APPROVED** [review]

**Comment by Sarrus1** [comment]
Thanks!

**Pull request merged**